ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/sflow.yml

```diff
@@ -24,3 +24,11 @@ sflow:
   sample: 1000
   dangerous: true
   run: true
+  hardware_acceleration:
+    enabled: true
+    sample: 1024
+    modules:
+      - name: Linecard2
+        enabled: true
```

Comment:
since `true` is the default, it would be nice to also test without the `enabled` key set.

Reply:
Fixed!